README.md

```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
 
 ```
 
+### Common Calls and Examples
+There are two main ways to use the client:
+1. Set the namespace
+2. Supply the namespace for calls that require it
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
+
+#### Create a Namespace
+``` 
+mc.create_namespace(namespace_name, 
+    "owner name",
+    "description of namespace")
+   ```
+
+#### Create a Job
+```
+mc.create_job(job_name=job_name,
+    location=job_location,
+    description=description,
+    input_dataset_urns=input_dataset_urns,
+    output_dataset_urns=output_dataset_urns,
+    namespace_name=namespace_name)
+```
+
+#### Create a JobRun
+```
+job_run_info = marquez_client.create_job_run(
```

Comment:
the client object is called `marquez_client` here, but everywhere else it is `mc`

Reply:
Good catch.